packages/bpk-component-calendar/src/BpkCalendarGrid-test.js

```diff
@@ -129,4 +129,29 @@ describe('BpkCalendarGrid', () => {
     expect(onDateClick.mock.calls.length).toBe(2);
     expect(onDateClick.mock.calls[1][0]).toEqual(new Date(2016, 9, 6));
   });
+
+  it('should call the onDateMouseDown callback', () => {
+    const onDateMouseDown = jest.fn();
+
+    const grid = mount(<BpkCalendarGrid
+      month={new Date('2016-10')}
+      formatMonth={formatMonth}
+      formatDateFull={formatDateFull}
+      DateComponent={BpkCalendarDate}
+      daysOfWeek={weekDays}
+      weekStartsOn={0}
+      showWeekendSeparator
+      onDateMouseDown={onDateMouseDown}
+    />);
+
+    expect(onDateMouseDown.mock.calls.length).toBe(0);
```

Comment:
This is redundant really

Reply:
You mean checking that the `length` is `0`? or the whole test?

Reply:
Checking that the length is zero. That's guaranteed by the mock implementation